config/routes.rb

```diff
@@ -69,7 +69,8 @@
 
   get '/patients/households/self_reporting', to: 'patients#self_reporting'
   get '/patients/:id/household_removeable', to: 'patients#household_removeable'
-  post '/patients/bulk_edit/status', to: 'patients#bulk_update_status'
+  post '/patients/bulk_edit/status', to: 'patients#bulk_update'
+  post '/patients/bulk_edit/assigned_user', to: 'patients#bulk_update'
```

Comment:
Would it be alright to just use `/patients/bulk_edit/status` for all the bulk edit update routes?

Reply:
If that's the case, should it be updated to be a more generic sounding route instead of one that mentions a specific data element. Would simply `/patients/bulk_edit` or `/patients/bulk_edit/patients_update` work?

Reply:
`/patients/bulk_edit` sounds ideal to me

Reply:
This update is now available.